### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by capJavert**
- [x] Build action
- [x] Setup env, tokens etc.
- [x] Deploy (commit to gh-pages branch)

### CODE REVIEW

**Comment by capJavert** [comment]
Resolves #2

**Comment by capJavert** [comment]
@choyiny hello, implementation is done. It does the same thing your travis build did.

Only one thing, for this to work you have to create personal access token (with repo access) and create secret `ACCESS_TOKEN` inside your repo settings.

Let me know if you need help with that, you can find some resources here:
* https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables
* https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          BRANCH: gh-pages # The branch the action should deploy to.
+          FOLDER: dist/ta # The folder the action should deploy.
+          CNAME: ta.choy.in
+          BUILD_SCRIPT: |
+            npm install
+            npm run build
+            cp dist/ta/index.html dist/ta/404.html
+
+on:
```
Comment (choyiny) [inline .github/workflows/deploy.yml]: As done in official workflows, put this at the top of the file

Comment (capJavert) [reply .github/workflows/deploy.yml]: Done.

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Build & Deploy
+        uses: JamesIves/github-pages-deploy-action@master
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```
Comment (choyiny) [inline .github/workflows/deploy.yml]: this should be `secrets.GITHUB_TOKEN`

**Pull request merged**